drivers/led/led_gpio.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT gpio_leds
+
+/**
+ * @file
+ * @brief GPIO driven LEDs
+ */
+
+#include <drivers/led.h>
+#include <drivers/gpio.h>
+#include <device.h>
+#include <zephyr.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(led_gpio, CONFIG_LED_LOG_LEVEL);
+
+#define DEV_CFG(dev)	((const struct led_gpio_config *) ((dev)->config))
```

Comment:
There's no need for this macro, just use `dev->config`

Reply:
ok, done